IPython/core/usage.py

```diff
@@ -93,42 +91,39 @@
 MAIN FEATURES
 -------------
 
-* Access to the standard Python help. As of Python 2.1, a help system is
-  available with access to object docstrings and the Python manuals. Simply
-  type 'help' (no quotes) to access it.
+* Access to the standard Python help with object docstrings and the Python
+  manuals. Simply type 'help' (no quotes) to invoke it.
 
 * Magic commands: type %magic for information on the magic subsystem.
 
 * System command aliases, via the %alias command or the configuration file(s).
 
 * Dynamic object information:
 
-  Typing ?word or word? prints detailed information about an object.  If
-  certain strings in the object are too long (docstrings, code, etc.) they get
-  snipped in the center for brevity.
+  Typing ?word or word? prints detailed information about an object.  Long
+  strings (docstrings, code, etc.) get snipped in the center for brevity.
 
   Typing ??word or word?? gives access to the full information without
-  snipping long strings. Long strings are sent to the screen through the less
-  pager if longer than the screen, printed otherwise.
+  snipping long strings. Strings that are longer than the screen are printed
+  through the less pager.
 
   The ?/?? system gives access to the full source code for any object (if
   available), shows function prototypes and other useful information.
 
   If you just want to see an object's docstring, type '%pdoc object' (without
   quotes, and without % if you have automagic on).
 
-* Completion in the local namespace, by typing TAB at the prompt.
+* Tab completion in the local namespace:
 
   At any time, hitting tab will complete any available python commands or
   variable names, and show you a list of the possible completions if there's
   no unambiguous one. It will also complete filenames in the current directory.
 
-* Search previous command history in two ways:
+* Search previous command history in multiple ways:
 
-  - Start typing, and then use Ctrl-p (previous, up) and Ctrl-n (next,down) to
-    search through only the history items that match what you've typed so
-    far. If you use Ctrl-p/Ctrl-n at a blank prompt, they just behave like
-    normal arrow keys.
+  - Start typing, and then use Ctrl-p (previous, up) and Ctrl-n (next, down)
+    to search through the history items that match what you've typed so
+    far. At a blank prompt, Ctrl-p/Ctrl-n behave like normal arrow keys.
```

Comment:
Actually, arrow keys now behave like Ctrl-p/Ctrl-n. This text hasn't been updated in a long time!